bare-metal/README.md

```diff
@@ -19,7 +19,9 @@ disk_size=80
 base_ip=50
 ssh_user=vagrant
 ram_size=4096
+cpus=1
 vm_base_name=mynode
+restart_vm_file_name=restart_vm
```

Comment:
Why the name of the restart script is a variable? 
Why don't just set the static name:`restart_vm`?